_content/av/buddisism-view-fortune-telling-fengshui_sheng-yen.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "How does Buddhism view the practice of fortune telling?"
+authors:
+  - sheng-yen
+external_url: "https://www.youtube.com/watch?v=kOrIWq0YgNw"
+drive_links:
+  - "https://drive.google.com/file/d/1B_12mi5l5UkduqmuplkCCuKd1fN4nppu/view?usp=drive_link"
+course: karma
+tags:
+  - chan
+  - mind
```

Comment:
maybe `problems` instead of mind?

Reply:
problems is cool. You'll have to explain to me once more what the mind tag is generally used for. In this short video, Sheng-yen spends time talking about cultivating the mind, etc.